<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewProgramDialog</class>
 <widget class="QDialog" name="NewProgramDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="titleWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="icons.qrc">:/icons/page_white_add.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="title_2">
        <property name="styleSheet">
         <string notr="true">QLabel { font-size: 14px; }</string>
        </property>
        <property name="text">
         <string>New Program</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>159</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="dialogMainWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="programDetailsGroup">
        <property name="title">
         <string>New Program Details</string>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="programNameLabel">
           <property name="text">
            <string>Program Name:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="programNameEdit"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="programDirLabel">
           <property name="text">
            <string>Program Directory:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <layout class="QHBoxLayout" name="programDirLayout">
           <item>
            <widget class="QLineEdit" name="programDirEdit"/>
           </item>
           <item>
            <widget class="QPushButton" name="programDirButton">
             <property name="text">
              <string>Select Directory</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="programPathLabel">
           <property name="text">
            <string>Program Path:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="programPath">
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>105</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="buttons" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>programNameEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>NewProgramDialog</receiver>
   <slot>nameChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>294</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>520</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>programDirEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>NewProgramDialog</receiver>
   <slot>dirChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>535</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>programDirButton</sender>
   <signal>pressed()</signal>
   <receiver>NewProgramDialog</receiver>
   <slot>selectDir()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>377</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>537</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewProgramDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>436</x>
     <y>355</y>
    </hint>
    <hint type="destinationlabel">
     <x>452</x>
     <y>412</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewProgramDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>356</x>
     <y>353</y>
    </hint>
    <hint type="destinationlabel">
     <x>360</x>
     <y>413</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>nameChanged(QString)</slot>
  <slot>selectDir()</slot>
  <slot>dirChanged(QString)</slot>
 </slots>
</ui>
